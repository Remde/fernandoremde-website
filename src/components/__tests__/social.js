import React from "react"
import renderer from "react-test-renderer"

import Social from "../social.js"

describe("Social", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(<Social />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
