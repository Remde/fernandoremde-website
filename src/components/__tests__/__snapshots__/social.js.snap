// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Social renders correctly 1`] = `
<div
  className="about"
  id="social-section"
>
  <a
    href="https://www.github.com/Remde"
    rel="noopener noreferrer"
    target="_blank"
  >
    <div
      className="github"
    />
  </a>
  <a
    href="mailto:ffremde@gmail.com"
  >
    <div
      className="mail"
    />
  </a>
  <a
    href="https://www.linkedin.com/in/fernando-remde-291a641a6/"
    rel="noopener noreferrer"
    target="_blank"
  >
    <div
      className="linkedin"
    />
  </a>
</div>
`;
